packages/@secretlint/secretlint-rule-no-dotenv/README.md

```diff
@@ -0,0 +1,31 @@
+# @secretlint/secretlint-rule-no-dotenv
+A secretlint rule for dotenv.
+
+Prevents commits  `.env` file because it may contain credentials.
+
+## Install
+
+Install with [npm](https://www.npmjs.com/):
+
+    npm install @secretlint/secretlint-rule-no-dotenv
+
+## Usage
+
+Via `.secretlintrc.json`(Recommended)
+
+```json
+{
+    "rules": [
+        {
+            "id": "@secretlint/secretlint-rule-no-dotenv"
+        }
+    ]
+}
+```
+
+## MessageIDs
+
+### FOUND_DOTENV_FILE
+> found .env file
+
+`.env` file was found.
```

Comment:
Can you add Author and LICENSE section?

e.g.) https://github.com/secretlint/secretlint/tree/master/packages/%40secretlint/secretlint-rule-secp256k1-privatekey#author

Reply:
Also add `LICENSE` file.

Secretlint adopt MIT license.

Reply:
@azu resolved by follows:

- 539f5591ab4d0c6dd7dd532e10a5301e13ae7a21
- 8f2877e0849ebc66eda1efb95d027d2344cd21e8
- cd773aeee5e8685e48bc9a7f72e2810956eb36fa